modules/ROOT/pages/salesforce/salesforce-connector-97.adoc

```diff
@@ -96,36 +95,33 @@ you can use with the Salesforce APIs:
-
-=== SOAP API
-
-The Salesforce http://www.salesforce.com/us/developer/docs/api/index.htm[SOAP API] provides secure access to your organization's information on Salesforce. Most of the operations that the Salesforce connector performs map to this API.
-
-All the Salesforce operations performed through the SOAP API have an optional parameter called "Headers" that can take any of the following https://developer.salesforce.com/docs/atlas.en-us.api.meta/api/soap_headers.htm[Salesforce SOAP Headers]:
-AllOrNoneHeader, AllowFieldTruncationHeader, AssignmentRuleHeader, CallOptions,
-EmailHeader, LocaleOptions, MruHeader, OwnerChangeOptions, QueryOptions,
-UserTerritoryDeleteHeader, DuplicateRuleHeader.
-
-=== Streaming API
-
-The Salesforce http://www.salesforce.com/us/developer/docs/api_streaming/[Streaming API] securely receives notifications for changes to your organization's information in Salesforce. See xref:salesforce/salesforce-connector-config-topics.adoc#receivedata[Receive Inbound Data From Salesforce] for more information on the use of the Streaming API.
+The Salesforce Connector works with the Salesforce SOAP API, REST API, Bulk API, and Streaming API, but does not provide access to the Chatter API or the Tooling API. Each API call uses an XML request and response over an HTTPS connection. All required request headers, error handling, and HTTPS connection configurations are built into the connector.
+
+* Apex SOAP API
++
+The Salesforce https://developer.salesforce.com/docs/atlas.en-us.apexcode.meta/apexcode/apex_api.htm?search_text=soap[Apex SOAP API] exposes Apex class methods as custom SOAP web service calls. This allows an external application to invoke an Apex web service to perform an action in Salesforce.
++
+* Apex REST API
++
+The Salesforce https://developer.salesforce.com/docs/atlas.en-us.apexcode.meta/apexcode/apex_rest.htm[Apex REST API] creates your own REST-based web services using Apex. This API has all of the advantages of the REST architecture, while adding the ability to define custom logic and including automatic argument or object mapping.
++
+* Bulk API
++
+The Salesforce https://developer.salesforce.com/docs/atlas.en-us.api_asynch.meta/api_asynch/asynch_api_intro.htm[Bulk API] quickly and securely loads batches of your organization's data into Salesforce. See also: xref:salesforce/salesforce-connector-config-topics.adoc#batchdata[Load Data in Batches].
++
+* Metadata API
++
+The Salesforce https://developer.salesforce.com/docs/atlas.en-us.api_meta.meta/api_meta/meta_intro.htm[Metadata API] manages customizations and build tools that work with the metadata model, not the data itself.
++
+* SOAP API
++
+The Salesforce https://developer.salesforce.com/docs/atlas.en-us.api.meta/api/sforce_api_quickstart_intro.htm[SOAP API] provides secure access to your organization's information on Salesforce. Most of the operations that the Salesforce Connector performs map to this API.

```

Comment:
GLOBALLY do not use "the" in front of connector names unless they are followed by a common noun.
```suggestion
The Salesforce https://developer.salesforce.com/docs/atlas.en-us.api.meta/api/sforce_api_quickstart_intro.htm[SOAP API] provides secure access to your organization's information on Salesforce. Most of the operations that the connector performs map to this API.
```
OR
```suggestion
The Salesforce https://developer.salesforce.com/docs/atlas.en-us.api.meta/api/sforce_api_quickstart_intro.htm[SOAP API] provides secure access to your organization's information on Salesforce. Most of the operations that Salesforce Connector performs map to this API.
```